### DISCUSSION TITLE
Pull request - CSS bug

### DISCUSSION THREAD

**Discussion by Pheat** [community/community#19345] [Projects and Issues] [answered]
Hey guys I don't know if this proper channel for report bugs but I found that in pull request review if you are going through changes and u will click on 3 dots on the end of the line then context menu will be hidden under title of changed file (screen)

![image](https://user-images.githubusercontent.com/9001387/175509959-3c4ef2f7-ed48-409a-9f22-40ab765f5a87.png)

**Comment by melvincwng** [comment, marked as answer]
@Pheat, thanks for highlighting this issue to the community. 

For bug reports, please kindly report them over [here](https://support.github.com/contact/bug-report).

Hope this clarifies.

If this has helped you, please 'Mark as answer'.

Thanks and have a nice day 😊

**Reply by Pheat** [reply]
Ou thank you. My bad I was not able to find better place where to report this bugs. :-) thank you.
